src/org/zaproxy/zap/extension/fuzz/httpfuzzer/processors/HttpFuzzerProcessorScript.java

```diff
@@ -33,12 +35,7 @@
  * @see HttpFuzzerTaskProcessorUtils
  */
 public interface HttpFuzzerProcessorScript {
-
-    /**
-     * The name used to identify the type of this script, for example, in configurations.
-     */
-    final static String TYPE_NAME = "httpfuzzerprocessor";
-
+    
```

Comment:
White space change should be reverted.

https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting